ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -75,19 +75,18 @@ export const ApplicationResourceList = ({
                                             <i className='fa fa-ellipsis-v' />
                                         </button>
                                     )}>
-                                    {() =>
-                                        nodeMenu({
-                                            name: res.name,
-                                            version: res.version,
-                                            kind: res.kind,
-                                            namespace: res.namespace,
-                                            group: res.group,
-                                            info: null,
-                                            uid: '',
-                                            resourceVersion: null,
-                                            parentRefs: []
-                                        })
-                                    }
+                                    {() => 
+                                         nodeMenu({
+                                        name: res.name,
+                                        version: res.version,
+                                        kind: res.kind,
+                                        namespace: res.namespace,
+                                        group: res.group,
+                                        info: null,
+                                        uid: '',
+                                        resourceVersion: null,
+                                        parentRefs: []
+                                    })}
```

Comment:
Is this change intentional?

Reply:
This change is not related to this PR. This file is in the same state as before. I didn't make any change in the code.